src/containers/Dashboard.vue

```diff
@@ -388,76 +387,8 @@ export default Vue.extend({
       return arr && arr.length !== 0 && arr[0] !== '' ? arr : ['N/A'];
     },
 
-    getCourseCodesArray(): readonly CourseTaken[] {
-      const courses: CourseTaken[] = [];
-      this.semesters.forEach(semester => {
-        semester.courses.forEach(course => {
-          const [subject, number] = course.code.split(' ');
-          courses.push({
-            code: `${course.lastRoster}: ${subject} ${number}`,
-            subject,
-            courseId: course.crseId,
-            number,
-            credits: course.credits,
-            roster: course.lastRoster,
-          });
-        });
-      });
-      courses.push(...getCourseEquivalentsFromUserExams(this.onboardingData));
-      return courses;
-    },
-
-    getRequirementTypeDisplayName(type: string): string {
-      return type.charAt(0).toUpperCase() + type.substring(1);
-    },
-
     recomputeRequirements(): void {
-      const groups = computeRequirements(
-        this.getCourseCodesArray(),
-        this.onboardingData.college,
-        this.onboardingData.major,
-        this.onboardingData.minor
-      );
-      // Turn result into data readable by requirements menu
-      const singleMenuRequirements = groups.map(group => {
-        const singleMenuRequirement: SingleMenuRequirement = {
-          ongoing: [],
-          completed: [],
-          name: `${
-            group.groupName.charAt(0) + group.groupName.substring(1).toLowerCase()
-          } Requirements`,
-          group: group.groupName.toUpperCase() as 'COLLEGE' | 'MAJOR' | 'MINOR',
-          specific: group.specific,
-        };
-        group.reqs.forEach(req => {
-          // Create progress bar with requirement with progressBar = true
-          if (req.requirement.progressBar) {
-            singleMenuRequirement.type = this.getRequirementTypeDisplayName(
-              req.requirement.fulfilledBy
-            );
-            singleMenuRequirement.fulfilled = req.totalCountFulfilled || req.minCountFulfilled;
-            singleMenuRequirement.required =
-              (req.requirement.fulfilledBy !== 'self-check' && req.totalCountRequired) ||
-              req.minCountRequired;
-          }
```

Comment:
now no requirement has progressbar any more. so it's removed. It's probably a relic of university requirements